patches/0003-Add-BoringSSL-crypto-backend.patch

```diff
@@ -30,15 +30,15 @@ index 00000000000000..0b62cef68546d0
 +var Dec = bbig.Dec
 diff --git a/src/crypto/internal/backend/boring_linux.go b/src/crypto/internal/backend/boring_linux.go
 new file mode 100644
-index 00000000000000..b22886bb6cc88f
+index 00000000000000..3b5504b6afc5c6
 --- /dev/null
 +++ b/src/crypto/internal/backend/boring_linux.go
 @@ -0,0 +1,135 @@
 +// Copyright 2017 The Go Authors. All rights reserved.
 +// Use of this source code is governed by a BSD-style
 +// license that can be found in the LICENSE file.
 +
-+//go:build goexperiment.boringcrypto && linux && cgo && amd64 && !android && !cmd_go_bootstrap && !msan
++//go:build goexperiment.boringcrypto && linux && cgo && (amd64 || arm64) && !android && !msan
```

Comment:
It looks like we might want to apply the platform change to 1.20 as well since this is included in some of those releases: https://github.com/golang/go/commit/39ec97da15e9167e7a17746c643f264baae6ce7b. However, doesn't seem like a priority because we don't anticipate anyone really depending on boring with our fork.

Reply:
Good catch, will do.